src/client/containers/FavoritePage/FavoritePage.Container.js

```diff
@@ -0,0 +1,52 @@
+import './FavoritePage.styles.css';
```

Comment:
There is a duplicate container with a typo the Favourite and Favorite. Can you delete the wrong one?

![image](https://user-images.githubusercontent.com/81110602/156635776-ebe4ff86-52d7-4b1c-8d3b-af093a3bb303.png)

Reply:
done